avc/src/start_detector/start_detector.cpp

```diff
@@ -0,0 +1,67 @@
+#include "start_detector.h"
+#include <pluginlib/class_list_macros.h>
+
+using namespace std;
+using namespace ros;
+using namespace cv;
+
+namespace avc {
+    const Scalar red_low{0, 0, 100};
```

Comment:
It'd be cool to name and parameterize these magic number constants (the color thresholds, Hough parameters, and sum threshold) so they can be easily tuned at competition if needed.